crates/ra_syntax/src/ast/generated.rs

```diff
@@ -50,10 +47,7 @@ impl std::fmt::Display for FnDef {
 }
 impl AstNode for FnDef {
     fn can_cast(kind: SyntaxKind) -> bool {
-        match kind {
-            FN_DEF => true,
-            _ => false,
-        }
+        matches!(kind, FN_DEF)
```

Comment:
I was thinking at one point about special-casing this with just `kind == FN_DEF`.